### PR TITLE
Layout exercise 

### PR DESCRIPTION
**Pull request by DavidDevJs**
Clon login page Twitter, CSS flexbox was applied and the other concepts were learned in the CSS module.

### CODE REVIEW

File: 7-layouts/layouts.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+                name="user"
+                placeholder="Phone, email, or username"
+                class="login-optional__form-input"
+              />
+              <input
+                type="password"
+                name="password"
+                placeholder="Password"
+                class="login-optional__form-input"
+              />
```
Comment (fabianmarinog) [inline 7-layouts/layouts.html:33]: Don't forget the <label> for each input

File: 7-layouts/styles.css [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+.footer__buttons .button--signup {
+  width: 150px;
+  margin-inline: 15px;
+}
+
+.footer__buttons .button--login {
+  width: 150px;
+  margin-inline: 15px;
+}
```
Comment (fabianmarinog) [inline 7-layouts/styles.css]:
```suggestion
.footer__buttons .button--signup,
.footer__buttons .button--login {
  width: 150px;
  margin-inline: 15px;
}
```

File: 7-layouts/styles.css [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    padding: 5px;
+  }
+
+  .login-optional__form-input::placeholder {
+    position: relative;
+    bottom: 11px;
+  }
+}
+
+@media screen and (min-width: 700px) {
```
Comment (fabianmarinog) [inline 7-layouts/styles.css:184]: This block should go in line 150, better to read from top to bottom what's being applied according to the media query.

File: 7-layouts/styles.css [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    height: 100vh;
+  }
+  .description {
+    width: 50vw;
+    height: 100vh;
+  }
+
+  .footer {
+    position: fixed;
+    bottom: 0px;
```
Comment (fabianmarinog) [inline 7-layouts/styles.css]:
```suggestion
    bottom: 0;
```

**Review by fabianmarinog — CHANGES_REQUESTED** [review]

File: 7-layouts/styles.css (hunk 3 above)
Comment (DavidDevJs) [reply 7-layouts/styles.css:184]: Taken note  👌

**Pull request merged**